tests/Microsoft.Identity.Web.Test/ClientAssertionTests.cs

```diff
@@ -15,14 +15,16 @@ public class ClientAssertionTests
         public async Task TestClientAssertion()
         {
             int n = 0;
-            ClientAssertionDescription clientAssertionDescription = new ClientAssertionDescription(
-                cancellationToken =>
+            ClientAssertionDescription clientAssertionDescription = new ClientAssertionDescription()
+            {
+                ClientAssertionProvider = (cancellationToken =>
```

Comment:
The ClientAssertionDescription class was internal and used to take the delegate in the constructor, but the delegate cannot be initialized form the derived classes, so I changed it to be a protected member, and updated the test